src/servers_non_y2/ag_corosync

```diff
@@ -0,0 +1,1055 @@
+#!/usr/bin/python3
+# -*- coding: utf8 -*-
+
+#
+# Authors:  Xinwei Hu <xwhu@suse.de>
+#    Lukas Ocilka <locilka@suse.cz>
+#
+# File:    ag_corosync
+#
+# License:
+#
+#   This program is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#   See the GNU General Public License for more details.
+#
+#   This program is free software; you can redistribute it and/or
+#   modify it under the terms of the GNU General Public License
+#   as published by the Free Software Foundation; either version
+#   2 of the License, or (at your option) any later version.
+#
+#   You should have received a copy of the GNU General Public License
+#   along with this program; if not, write to the Free Software
+#   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+#   02111-1307 USA
+#
+
+import gettext, os, re
+from gettext import textdomain
+textdomain("corosync")
```

Comment:
All the other openais -> corosync changes are contained within this repo, but the textdomain is not. Is there a matching PR to change it in the translations repo? Perhaps it is easier to keep openais here?

Reply:
In principle you are right, but there is no text to translate in this file. I.e. the entry textdomain("corosync") itself is superfluous. I'll remove it.

Reply:
See next commit.